src/lib.rs

```diff
@@ -0,0 +1,20 @@
+#[macro_use]
```

Comment:
https://doc.rust-lang.org/edition-guide/rust-2018/macros/macro-changes.html seems to say that #[macro_use] is no longer required with Rust Edition 2018.

Reply:
Changed it to explicit use statements.